envoy/service/discovery/v2/ads.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package envoy.service.discovery.v2;
+
+import "envoy/service/discovery/v2/common.proto";
+
+// [#not-implemented-hide:] Not configuration. Workaround c++ protobuf issue with importing
```

Comment:
Can you file an issue on protobuf GH and include it in all these comments, together with a TODO to cleanup at some point?

Reply:
filed https://github.com/golang/protobuf/issues/494

Reply:
https://github.com/google/protobuf/issues/4221